extensions/interactions/MathExpressionInput/directives/MathExpressionInput.js

```diff
@@ -130,30 +135,51 @@ oppia.directive('oppiaInteractiveMathExpressionInput', [
               setGuppyContentFromInput();
             });
           };
-
-          var guppyInstance = new Guppy(guppyDivElt, {
-            empty_content: (
+          var guppyInstance = new Guppy(guppyDivId, {
+            settings: {
+              empty_content: (
               '\\color{grey}{\\text{\\small{Type a formula here.}}}'),
-            ready_callback: function() {
-              Guppy.get_symbols(
-                UrlInterpolationService.getStaticAssetUrl(
-                  '/overrides/guppy/oppia_symbols.json'));
-
-              if (DeviceInfoService.isMobileUserAgent() &&
-                DeviceInfoService.hasTouchEvents()) {
-                $scope.mobileOverlayIsShown = true;
-                // Wait for the scope change to apply. Since we interact with
-                // the DOM elements, they need to be added by angular before
-                // the function is called. Timeout of 0 to wait until the end
-                // of the current digest cycle, false to not start a new digest
-                // cycle. A new cycle is not needed since no angular variables
-                // are changed within the function.
-                $timeout(makeGuppyMobileFriendly, 0, false);
+              buttons: []
+            },
+            events: {
+              done: function(e) {
```

Comment:
Is this called every time enter key is pressed? If so is this desirable, @seanlip?

Reply:
Yes. I have tried to imitate the same functionality as implemented previously by this code:

'''
 guppyInstance.done_callback = function() {
            $scope.submitAnswer();
          };
'''

Reply:
Was the previous done_callback() also called every time the Enter key is pressed? If so then preserving that behaviour is fine.

Reply:
I rechecked and Yes it is called each time enter key is pressed.

Reply:
OK, I think it's fine to preserve the behaviour then!

Reply:
Thanks for confirming @paroothisumit This looks fine.